includes/Core/Util/Tracking.php

```diff
@@ -194,14 +207,19 @@ private function inline_js_admin_data( $data ) {
 	 * Register tracking settings and allow access from Rest API.
 	 *
 	 * @since 1.0.0
+	 * @since n.e.x.t Registers a meta field instead of setting.
 	 */
 	private function register_settings() {
+		global $wpdb;
 		$args = array(
 			'type'         => 'boolean',
 			'description'  => __( 'Allowing tracking of anonymous usage stats.', 'google-site-kit' ),
 			'default'      => false,
-			'show_in_rest' => true,
+			'single'       => true,
+			'show_in_rest' => current_user_can( Permissions::SETUP ),
 		);
-		register_setting( 'google-site-kit', self::TRACKING_OPTIN_KEY, $args );
+		// Need to conditionally include the blog prefix as this is a user option.
+		$prefix = ! $this->context->is_network_active() ? $wpdb->get_blog_prefix() : '';
```

Comment:
This should be using `Context::is_network_mode()` instead.

Reply:
```suggestion
		$prefix = ! $this->context->is_network_mode() ? $wpdb->get_blog_prefix() : '';
```